src/local-editor-service.ts

```diff
@@ -295,6 +327,30 @@ export class LocalEditorService extends EditorService {
 
       return concatMap(elements, (el) => el.attributes)
           .find(at => at.name === location.attribute);
+    } else if (location.kind === 'attributeValue' || location.kind === 'text') {
+      const domModules = document.getByKind('dom-module');
+      for (const domModule of domModules) {
+        const elements =
```

Comment:
Could you somehow query the analyzer for the Polymer elements it has presumably already discovered looking at this file? Surprised to see re-scanning the HTML here.